config/aic.php

```diff
@@ -42,8 +42,12 @@
 
     // Branding
     'branding' => [
-        'digital_publications' => [
-            'colors' => [
+        'colors' => [
+            'general' => [
+                '#1E3F49',
```

Comment:
@nikhiltri I was unable to find this color in the DB, but I included it here under "general" colors.

Reply:
I queried `select * from blocks where type = 'custom_banner'` and eyeballed the results. It looks like the one record with a color defined has a `background_type` of `background_image`. Maybe it used to be a solid color, but they've since changed it.